src/main/java/net/mcreator/ui/blockly/BlocklyJavascriptBridge.java

```diff
@@ -138,6 +139,24 @@
 		callback.call("callback", StringUtils.join(retval, ','));
 	}
 
+	@SuppressWarnings("unused") public void openArgExecuteProcedureEditor(String data, JSObject callback) {

```

Comment:
You should use existing "call procedure block" for this. Add this block to your toolbox and drop setting execute procedure with this so we don't duplicate existing functionality if possible.

Reply:
Simple questio. How do you want to use all dependencies provided by the command context (including arguments) without adding a new block or a new interface like you made for AI conditons?

Reply:
Call procedure block automatically loads all dependencies required by the calling procedure so if this procedure requires arguments, they will be added to the dependency list